backend/chainlit/sync.py

```diff
@@ -10,9 +10,8 @@
 import threading
 
 from asyncer import asyncify
-from syncer import sync
-
 from chainlit.context import context
+from syncer import sync
```

Comment:
wrong

Reply:
This was moved automatically, but I'm curious, why is that wrong?